frontend/micro-ui/web/micro-ui-internals/packages/css/src/home/home.scss

```
@@ -2044,3 +2044,109 @@
     justify-content: center;
   }
 }
+
+@media (max-width: 480px) {
```

Stylelint (17.2.0)

[error] Expected "context" media feature range notation (media-feature-range-notation)

(media-feature-range-notation)